build/ci/postInstall.js

```diff
@@ -76,5 +77,23 @@ function createJupyterKernelWithoutSerialization() {
     console.log(colors.green(destPath + ' file generated (by Jupyter VSC)'));
 }
 
+/**
+ * In order to generate random bytes on Windows without taking a dependency on native node modules
+ * which we then need to build xplat and bundle with the extension, download a prebuilt executable
+ * which directly consumes BCryptGenRandom in bcrypt.dll and outputs random bytes as hex. This
+ * executable is required for trusted notebooks key generation and is included with the built extension.
+ */
+function downloadBCryptGenRandomExecutable() {
+    console.log('Downloading BCryptGenRandom.exe...');
+    const executableName = 'BCryptGenRandom.exe';
+    const uri = `https://pvsc.blob.core.windows.net/extension-builds-juypter/${executableName}`;
```

Comment:
Hmm, this would likely make me nervous if I was outside of Microsoft. I'm trusting this executable to be downloaded and run on my machine. How do I know this blob storage site is secure? What if somebody hacks our blob storage?

Trying to think of a better solution but not having much luck at the moment.

Reply:
I wonder if we could publish the exe to the vscode marketplace?

Reply:
As a separate extension?

Reply:
Oh wait never mind. This isn't happening at run time. It's happening during build time. That seems fine to me.